apps/block_scout_web/assets/js/lib/market_history_chart.js

```diff
@@ -114,7 +116,7 @@ class MarketHistoryChart {
     this.marketCap = {
       label: window.localized['Market Cap'],
       yAxisID: 'marketCap',
-      data: getMarketCapData(marketHistoryData, availableSupply),
+      data: [], // getMarketCapData(marketHistoryData, availableSupply),
```

Comment:
Am I right assuming that this (and lines 135 + 137) means we do not show any marketcap information anymore but only the cGLD/USD price?

Reply:
It still shows the marketcap below the graph, next to Price. Should we remove that too?

Reply:
No I think that is fine. All else LGTM!